src/System.Net.HttpListener/tests/GetContextHelper.cs

```diff
@@ -59,7 +59,7 @@ public class CustomAsyncResult : IAsyncResult
     public static class Helpers
     {
         public static bool IsWindowsImplementation { get; } = PlatformDetection.IsWindows && PlatformDetection.IsNotOneCoreUAP;
-        public static bool IsNotWindowsImplementation => !IsWindowsImplementation;
+        public static bool IsNotWindowsImplementation => !IsWindowsImplementation && PlatformDetection.IsNotOneCoreUAP;
```

Comment:
@CIPop Are you okay with my current change (to add IsNotOneCoreUAP to the IsNotWindows check?) It should be targeted enough.